src/control/cmd/daos/system.go

```diff
@@ -0,0 +1,71 @@
+//
+// (C) Copyright 2023 Intel Corporation.
+//
+// SPDX-License-Identifier: BSD-2-Clause-Patent
+//
+
+package main
+
+/*
+#include "util.h"
+*/
+import "C"
+
+import (
+	"unsafe"
+
+	"github.com/pkg/errors"
+)
+
+type rankURI struct {
+	Rank int    `json:"rank"`
```

Comment:
minor: We use `uint32` for ranks everywhere else in the codebase. This struct isn't exported outside of the daos tool, so it's not a big deal, but in general we should try to be consistent and also avoid type conversions unless necessary.

Reply:
I'll address this since I need to restart the CI job anyway.